tests/unit_tests/scheduler/test_openpbs_driver.py

```diff
@@ -203,20 +203,22 @@ async def test_cluster_label():
     "Job id            Name             User              Time Use S Queue\n"
     "----------------  ---------------- ----------------  -------- - -----\n"
 )
+"Job id            Name             User              Time Use S Queue\n----------------  ---------------- ----------------  -------- - -----"
+formatter = "%-30s %-15s %-15s %-8s %-1s %-5s"
 
 
 @pytest.mark.parametrize(
     "qstat_script, started_expected",
     [
         pytest.param(
-            f"echo '{QSTAT_HEADER}1 foo someuser 0 R normal'; exit 0",
+            f"echo '{QSTAT_HEADER}';printf '{formatter}' 1 foo someuser 0 R normal; exit 0",
```

Comment:
Is there a better way of having it formatted the same way as the actual qstat command with -w set?

Reply:
Since QSTAT_HEADER is the short header, this test is generating an impossible qstat output, isn't it? (our code probably handles that fine, but it is not an important feature to support, so we don't need to test that weird combination - just change QSTAT_HEADER to the wide format).

Reply:
I ran `qstat -w` while having an active job, and copied the header directly from there.